src/config/index.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const fs = require('fs');
+const Path = require('path');
+
+const getSettingsObject = (settings) => {
+  try {
+    return JSON.parse(settings);
+  } catch (e) {
+    return null;
+  }
+};
+
+const getSettingsFromFile = (settingsPath) => {
+  if (fs.existsSync(settingsPath)) {
```

Comment:
should this be statSync().isFile() as it will return successfully for that if its just the path?

Reply:
Good point. though docs actually say not to use stat of existence checks before fileRead. So I will just remove it